app/models/foreman_tasks/task.rb

```diff
@@ -158,6 +158,25 @@ def add_missing_task_groups(groups)
       (groups - task_groups).each { |group| task_groups << group }
     end
 
+    def sub_tasks_counts
+      success   = sub_tasks.where(:result => 'success').count
+      pending   = sub_tasks.where(:result => 'pending').count
+      failed    = sub_tasks.where(:result => %w(error warning)).count
+      cancelled = sub_tasks.where(:result => 'cancelled').count
```

Comment:
This is not working fully with batch planning of subtasks: the chart in rex looks like window's file copy dialog https://xkcd.com/612/ :). We need to include:

1. the count of unplanned tasks in in :pending when the task is not stopped
2. the count of unplanned tasks in :cancelled, if the task is stopped